boards/f4vi1/include/periph_conf.h

```diff
@@ -104,6 +104,17 @@ static const uart_conf_t uart_config[] = {
 #define UART_NUMOF          (sizeof(uart_config) / sizeof(uart_config[0]))
 /** @} */
 
+/**
+ * @brief   ADC configuration
+ *
+ * HACK: remove once periph drivers are only compiled if their FEATURE is set
+ * @{
+ */
+static const adc_conf_t adc_config[];
```

Comment:
That should not work (and murdock is already complaining). A c-file including this array can't know of it's size... Also: static makes this local to the compile unit so any  C file including this header generates its own array.